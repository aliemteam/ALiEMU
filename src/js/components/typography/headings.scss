@import 'variables';

.sectionHeading {
    margin: 0 {
        bottom: 10px;
    }
    font-size: 16px;
    font-family: $font-mono;
    font-weight: 500;
    opacity: 0.8;
}
