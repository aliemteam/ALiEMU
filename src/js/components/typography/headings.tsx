import React from 'react';

import styles from './headings.scss'

export const SectionHeading = (props: React.HTMLProps<HTMLHeadingElement>) => (
    <h2 {...props} className={styles.sectionHeading} />
);
