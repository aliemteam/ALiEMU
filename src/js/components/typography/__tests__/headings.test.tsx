import React from 'react';
import renderer from 'react-test-renderer';

import { SectionHeading } from '../headings';

it('should render <SectionHeading>', () => {
    const tree = renderer.create(<SectionHeading>Hello World</SectionHeading>);
    expect(tree).toMatchSnapshot();
});
