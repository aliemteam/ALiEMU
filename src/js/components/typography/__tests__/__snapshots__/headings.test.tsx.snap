// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render <SectionHeading> 1`] = `
<h2
  className="sectionHeading"
>
  Hello World
</h2>
`;
